lib/edit/p_info.txt

```diff
@@ -173,8 +171,8 @@ C:a:k:=0:=0:Axe-mastery
 C:a:k:=0:=0:Hafted-mastery
 C:a:k:=0:=0:Polearm-mastery
 C:a:k:+1000:-200:Archery
-C:a:k:+1000:+900:Sneakiness
-C:a:k:+1000:+900:Disarming
+
```

Comment:
These newlines should be removed -- ditto for everywhere else in this file + corresponding Theme file.